src/decision/src/GpsManager.cpp

```diff
@@ -34,12 +34,12 @@ GpsManager::GpsManager(int argc, char **argv, std::string node_name){
 
     // Setup Subscribers
     uint32_t refresh_rate = 10;
-    std::string raw_gps_topic_name = "/gps_driver/gps";
+    std::string raw_gps_topic_name = "/gps_driver/navsatfix";
     raw_gps_subscriber = public_nh.subscribe(raw_gps_topic_name, refresh_rate,
```

Comment:
Yeah, that should be private.... it is actually a private namespace, the variable name is just wrong, see #74 for more details..... in any case, changed.